### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by rmohr**
* Document which digests to use.
* Update all docker references from`rmohr/` to `kubevirtci/`
* Rename `kubeadm-1.9.3` cluster to `k8s-1.9.3`

Signed-off-by: Roman Mohr <rmohr@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,13 @@
 * `base` contains the base image with some scripts, qemu and dnsmasq
 * `centos7` adds a vagrant centos7 box to the image
 * `cli` contains a tool for provisioning, running and managing the containerized clusters
-* `kubeadm-1.9.3` k8s-1.9.3 cluster based on the centos7 image, provisioned with kubeadm
+* `k8s-1.9.3` k8s-1.9.3 cluster based on the centos7 image, provisioned with kubeadm
+* `os-3.9` os-3.9 cluster based on the centos7 image, provisioned with openshift-ansible
+
+## Versions to use
+
+* `kubevirtci/cli`: `sha256:e640c6a23e75eeb98ac8230c0542c83a0a4a556bed72d99fa89b3fb68c3976d5`
```
Comment (rmohr) [inline README.md:11]: @cynepco3hahue that is already the one with nfs.

Comment (cynepco3hahue) [reply README.md:11]: Will update my PR

File: base/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
-FROM fedora:27
+#FROM fedora:27
```
Comment (cynepco3hahue) [inline base/Dockerfile:1]: Do you still need it or we can just remove it?

File: cli/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,5 @@
-FROM fedora:27
+#FROM fedora:27
```
Comment (cynepco3hahue) [inline cli/Dockerfile:1]: The same as above

**Review by cynepco3hahue — COMMENTED** [review]
Two small comments

File: base/Dockerfile (hunk 2 above)
Comment (rmohr) [reply base/Dockerfile:1]: I just kept it to indicate that we are in general using fedora version 27.

**Pull request merged**